lms/djangoapps/branding/api.py

```diff
@@ -569,22 +568,43 @@ def get_logo_url(is_secure=True):
         is_secure (bool): If true, use HTTPS as the protocol.
     """
 
-    # if the configuration has an overide value for the logo_image_url
+    # if the configuration has an override value for the logo_image_url
     # let's use that
-    image_url = configuration_helpers.get_value('logo_image_url')
-    if image_url:
-        return _absolute_url_staticfile(
-            is_secure=is_secure,
-            name=image_url,
-        )
+    logo_url = settings.LOGO_URL
+    default_local_path = 'images/logo.png'
+    logo_image_path = configuration_helpers.get_value('logo_image_url')
+
+    if logo_image_path:
+        return _absolute_url_staticfile(is_secure=is_secure, name=logo_image_path)
 
     # otherwise, use the legacy means to configure this
     university = configuration_helpers.get_value('university')
-
     if university:
         return staticfiles_storage.url('images/{uni}-on-edx-logo.png'.format(uni=university))
-    else:
-        return staticfiles_storage.url('images/logo.png')
+
+    if logo_url:
+        return logo_url
+
+    return staticfiles_storage.url(default_local_path)
+
+
+def get_favicon_url():
+    """
+    Notes -- to be replaced with docstring.
+    1. Try to get path from config (favicon_path)
+    2. Return favicon_url if set in settings.
+    3. return local favicon from localstorage images/favicon.ico
+    """
+    favicon_url = settings.FAVICON_URL
+    default_local_path = settings.FAVICON_PATH
```

Comment:
Is `FAVICON_PATH` an existing setting? 

By the way, the difference between URL and PATH can be confusing.

Reply:
`FAVICON_PATH` is the existing settings, and `FAVICON_URL` is our new setting. yes it a little confusing.